assets/js/app.js

```diff
@@ -135,13 +140,13 @@ var segAvgRating = 0;
 var segIndexVotes = 0;
 var segIndexPrice = 0;
 var segIndexRating = 0;
+var rankIndex;
+var highestRanked_resto_id;
 
 $(document).ready(function () {
     
     $("#submit-btn").on("click", (event) => {
     event.preventDefault();
-    city1 = $("#startLocation").val().trim();
```

Comment:
city1 = $("#startLocation").val().trim();
    city2 = $("#endLocation").val().trim();

these need to be added back in before this can be approved # #49